src/PostalCode.vue

```diff
@@ -1,16 +1,14 @@
 <template>
   <stripe-element
+    ref='element'
     type='postalCode'
     :stripe='stripe'
     :value='value'
     :options='options'
     @blur='$emit("blur")'
     @focus='$emit("focus")'
     @empty='$emit("empty")'
-    @complete='$emit("complete")'
-    @brand='$emit("brand", $event)'
-    @error='$emit("error", $event)'
-    @value='$emit("value", $event)'
```

Comment:
@nkoehring, do you remember why does PostalCode.vue contain there listeners?
```
@complete='$emit("complete")'
@brand='$emit("brand", $event)'
@error='$emit("error", $event)'
@value='$emit("value", $event)'
```

I like the idea of replacing them with single `@change` but not sure if it is safe for those who will update the module

Reply:
@softbeehive this was just to bring them inline with the other components, which all use the same `change` event. Based on the commit message it seemed like they all used to have multiple events and this one was just missed?

Reply:
that's what it seems to me too, will test it and release if all good

Reply:
I remembered I could check git logs too. This looks like where they got out of sync. https://github.com/fromAtoB/vue-stripe-elements/commit/5218fbce5766741c58a9809ad8f1863619800e31#diff-25d902c24283ab8cfbac54dfa101ad31